controlers/topic-controler.js

```diff
@@ -0,0 +1,13 @@
+const { fetchTopics, invalidPaths } = require("../models/topic-model");
+
+exports.getTopics = (req, res, next) => {
+  fetchTopics().then((results) => {
+    res.status(200).send({ topics: results });
+  });
+};
+
+exports.invalidRoutes = (req, res, next) => {
+  invalidPaths().catch((err) => {
+    next(err);
+  });
+};
```

Comment:
I really like that this has been added, but it shouldnt exist in this topic-controller file.